README.md

```diff
@@ -15,15 +15,17 @@ A crawler that provides timely response to data change on public rental house pl
 
 ## Setup
 
-### Using Docker Env
+choose one of the options below (Docker or virtualenv) 
+
+#### If you're gonna work in `Docker` Env
```

Comment:
Let's do formal writing as it's the style of this README.

1. Avoid using `you` when possible. [reference](http://en.writecheck.com/blog/2012/11/30/writing-no-no-1-video)
2. Avoid slang.

Reply:
thanks for the suggestion!